analyses/transcriptomic-dimension-reduction/01-dimension-reduction.sh

```diff
@@ -73,7 +73,24 @@ Rscript --vanilla scripts/run-dimension-reduction.R \
   --neighbors ${NEIGHBORS} \
   --low_count_threshold ${COUNT_THRESHOLD} \
   --log2_transform
+  
+  
 
+#### RSEM log2 stranded, excluding mitochondrial genes and skipping t-sne ------------------------------------
+# Note that these results are only used for exploration in `05-seq_center-mitochondrial-genes.Rmd`
+Rscript --vanilla scripts/run-dimension-reduction.R \
+  --expression ../../data/pbta-gene-expression-rsem-fpkm.stranded.rds \
+  --metadata ${METADATA} \
+  --filename_lead rsem_stranded_no-mito_log \
```

Comment:
I see here that the script is saving as `no-mito`, but I don't see any option where it is actually excluding mitochondrial genes. Am I missing something?

Maybe?
```suggestion
  --remove_mito_genes \
  --filename_lead rsem_stranded_no-mito_log \
```

Reply:
the return of the flag!